### PR TITLE
View pages

### PR DESCRIPTION
**Pull request by litongz73**
add a connection with the algorithm

### CODE REVIEW

**Review by MinhDang98 — APPROVED** [review]

**Review by John-STJB — APPROVED** [review]

**Review by henrywang1 — APPROVED** [review]

File: zlp-scheduler/app/views/admin/view_result.html.haml [hunk 1]
```diff
@@ -11,37 +11,40 @@
...
           %form
-            %button.choosebtn{:formaction => "view_term_admin"} Choose
+            %button.choosebtn{:formaction => "view_conflicts"} Choose
       %tr
         %td#special_table_data Friday
         %td#special_table_data 1:45 PM - 3:45 PM
         %td#special_table_data
           %form
-            %button.choosebtn{:formaction => "view_term_admin"} Choose
+            %button.choosebtn{:formaction => "view_conflicts"} Choose
```
Comment (BWitemeyer) [inline zlp-scheduler/app/views/admin/view_result.html.haml:33]: Can we get rid of the hard-coded table entries? And just have the ones from the database?

Comment (BWitemeyer) [reply zlp-scheduler/app/views/admin/view_result.html.haml:33]: Both here and below, in the conflicts table.

**Review by BWitemeyer — CHANGES_REQUESTED** [review]
Please see the comments about taking out the hard-coded results.

**Comment by BWitemeyer** [comment]
Are there any tests written for this code? If not, we can make another user story for adding them.

**Review by BWitemeyer — APPROVED** [review]

**Review by adauzat90 — APPROVED** [review]

**Pull request merged**